doc/releases/changelog-0.11.0.md

```diff
@@ -202,79 +266,157 @@
   for the `StatePrep`, `BasisState`, and `QubitUnitary` operations. For the parameter-shift method,
   this is allowable for all operations.
 
-* Changes to support a dynamic number of qubits:
-
-  - The `qalloc_p` custom JAX primitive can now take in a dynamic number of qubits as a tracer
-    and lower it to mlir.
-    [(#1549)](https://github.com/PennyLaneAI/catalyst/pull/1549)
-
-  - `ComputationalBasisOp` can now take in a quantum register in mlir, instead of an explicit, fixed-size list of qubits.
-    [(#1553)](https://github.com/PennyLaneAI/catalyst/pull/1553)
+* An `mlir_opt` property has been added to `qjit` to access the optimized MLIR representation of a compiled 
```

Comment:
@dime10 what's the difference between this and `qjit.mlir`? Is `mlir_opt` simply the output MLIR post optimizations?

Reply:
`qjit.mlir` is the IR the Catalyst core receives as input, and `qjit.mlir_opt` it produces as output, that is after running everything in the MLIR stage of the entire pipeline.